TangoTest.cpp

```diff
@@ -1733,6 +1741,44 @@ void TangoTest::write_ulong_scalar(Tango::WAttribute &attr)
 	/*----- PROTECTED REGION END -----*/	//	TangoTest::write_ulong_scalar
 }
 //--------------------------------------------------------
+/**
+ *	Read attribute enum_scalar related method
+ *	Description: 
+ *
+ *	Data type:	Tango::DevEnum (enum_scalarEnum)
+ *	Attr type:	Scalar
+ */
+//--------------------------------------------------------
+void TangoTest::read_enum_scalar(Tango::Attribute &attr)
+{
+	DEBUG_STREAM << "TangoTest::read_enum_scalar(Tango::Attribute &attr) entering... " << std::endl;
+	/*----- PROTECTED REGION ID(TangoTest::read_enum_scalar) ENABLED START -----*/
+	//	Set the attribute value
+	attr.set_value(attr_enum_scalar_read);
+	
+	/*----- PROTECTED REGION END -----*/	//	TangoTest::read_enum_scalar
+}
+//--------------------------------------------------------
+/**
+ *	Write attribute enum_scalar related method
+ *	Description: 
+ *
+ *	Data type:	Tango::DevEnum (enum_scalarEnum)
+ *	Attr type:	Scalar
+ */
+//--------------------------------------------------------
+void TangoTest::write_enum_scalar(Tango::WAttribute &attr)
+{
+	DEBUG_STREAM << "TangoTest::write_enum_scalar(Tango::WAttribute &attr) entering... " << std::endl;
+	//	Retrieve write value
+	enum_scalarEnum	w_val;
+	attr.get_write_value(w_val);
+	/*----- PROTECTED REGION ID(TangoTest::write_enum_scalar) ENABLED START -----*/
+  *attr_enum_scalar_read = w_val;
```

Comment:
Indentation here is inconsistent with the other code in this function and with code in corresponding read_ function.